certified-connectors/ActiveCampaign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Enable integrations with your ActiveCampaign account using Rest APIs.",
```

Comment:
@mmchome Please remove the word "API" from the description. please follow our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
done